olm_deploy/operatorregistry/Dockerfile

```diff
@@ -11,6 +11,6 @@ RUN chmod -R g+w /manifests && rm /manifests/0*.yaml /manifests/art.yaml /manife
 
 COPY olm_deploy/scripts/registry-init.sh /scripts/
 
-COPY --from=registry-builder /build/bin/initializer /usr/bin/initializer
-COPY --from=registry-builder /build/bin/registry-server /usr/bin/registry-server
+COPY --from=registry-builder /bin/initializer /usr/bin/initializer
```

Comment:
nit -- you can probably rebase to drop this file's changes since we already fixed it